xchange-bittrex/src/main/java/org/knowm/xchange/bittrex/v1/BittrexUtils.java

```diff
@@ -26,10 +31,13 @@ public static String toPairString(CurrencyPair currencyPair) {
     return currencyPair.counter.getCurrencyCode().toUpperCase() + "-" + currencyPair.base.getCurrencyCode().toUpperCase();
   }
 
-  public static Date toDate(String date) {
-    Calendar cal = DatatypeConverter.parseDateTime(date);
-    cal.setTimeZone(TimeZone.getTimeZone(TIMEZONE));
-    return cal.getTime();
+  public static Date toDate(String dateString) {
+    try {
+      return DATE_PARSER.parse(dateString);
+    } catch (ParseException e) {
+      e.printStackTrace();

```

Comment:
I think it would be best to use a SLF4J logger here rather than print to stderr.